runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -0,0 +1,224 @@
+package org.corfudb.runtime.collections;
```

Comment:
Can you also implement a close that runtime calls on shutdown ?

Reply:
currently runtime has no visibility into CorfuStore, since CorfuStore runs above.
need to figure out a way to tie those two together. if you have any ideas on this, please share.

Reply:
I see, but we still need a close method, so that the consumer can explicitly call close if not the our runtime.

Reply:
tied up the runtime.close() to shutdown the executor pool.